docs/getting_started/advanced.ipynb

```diff
@@ -227,17 +224,17 @@
     "        },\n",
     "        every_n_iter=100,\n",
     "        use_graphite=False,\n",
-    "        use_mlflow=False\n",
+    "        use_mlflow=use_mlflow # change it to False if you don't want to track it in mlflow\n",
```

Comment:
set `USE_MLFLOW = False` at the beginning of the notebook and set all `use_mlflow=USE_MLFLOW`